notifications/src/main/java/org/openedx/notifications/presentation/inbox/NotificationsInboxViewModel.kt

```diff
@@ -111,6 +115,15 @@ class NotificationsInboxViewModel(
         internalLoadNotifications()
     }
 
+    fun updateNotifications() {
```

Comment:
Can we combine both methods?
`onReloadNotifications()` & `updateNotifications()`,

Reply:
For this, I would need to pass a variable, which, in my opinion, often confuses developers about its purpose. Let’s avoid doing that this time.

Reply:
Renmaed it to `onRefreshNotifications` as you suggested. 👍🏻